compiler/testData/diagnostics/testsWithStdLib/contracts/dsl/errors/contractCallSites.1.3.fir.kt

```diff
@@ -56,7 +56,7 @@ val topLevelAnonymousFunction = fun (x: Boolean) {
     contract { returns() implies (x) }
 }
 
-var topLevelPropertyAccessors: Int? = 42
+var topLevelPropertyAccessors: Int? = <!PROPERTY_INITIALIZER_NO_BACKING_FIELD!>42<!>
```

Comment:
This and below are false alarms, but mainly due to the unresolved backing field. Filed https://youtrack.jetbrains.com/issue/KT-44681

Reply:
3rd commit adds a local scope for backing fields before resolving property accessors, and now they're properly resolved. Also, these false alarms are gone too.

Reply:
According to @demiurg906, this is by design: https://youtrack.jetbrains.com/issue/KT-44681#focus=Comments-27-4674827.0-0 I'll revert the 3rd commit.